08.ls_object/file_info.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require './08.ls_object/detail'
+
+require 'etc'
+
+class FileInfo
+  def initialize(files)
+    @files = files
+  end
+
+  def build
+    file_details = build_file_details
+
+    file_details.map do |file_detail|
+      detail = Detail.new(file_detail)
+      row_data = []
+      row_data << detail.build_permission
+      row_data << file_detail[:nlink].to_s.rjust(3, ' ')
+      row_data << Etc.getpwuid(file_detail[:uid]).name.ljust(15, ' ')
+      row_data << Etc.getgrgid(file_detail[:gid]).name
+      row_data << file_detail[:size].to_s.rjust(5, ' ')
+      row_data << detail.build_updated_at
+      row_data << file_detail[:file_name]
+      row_data
```

Comment:
こんな感じで書くとすこしだけコンパクトにできるかなと思いました。（動作未検証です）

```suggestion
      [
        detail.build_permission,
        file_detail[:nlink].to_s.rjust(3, ' '),
        Etc.getpwuid(file_detail[:uid]).name.ljust(15, ' '),
        Etc.getgrgid(file_detail[:gid]).name,
        file_detail[:size].to_s.rjust(5, ' '),
        detail.build_updated_at,
        file_detail[:file_name]
      ]
```